pages/sales-pos/index.vue

```diff
@@ -1,109 +1,235 @@
 <template>
   <PageContainer :key="route.path">
-    <PageHeader title="LikerLand Book Store POS" />
-    <PageBody class="flex flex-col items-stretch grow space-y-4">
-      <UCard>
-        <UFormGroup label="Edit">
-          <UButton @click="toggleEditMode">
-            Toggle Edit Mode
-          </UButton>
-        </UFormGroup>
-        <UFormGroup v-if="isEditMode" label="Enter NFT Class ID">
-          <UInput
-            v-model="newProductId"
-            class="font-mono"
-            placeholder="likenft....,https://liker.land/nft/class/likenft...."
+    <PageHeader title="Liker Land Bookstore POS" />
+    <PageBody
+      :ui="{
+        base: 'flex flex-col items-stretch grow space-y-4',
+        padding: 'max-md:!px-0 max-md:py-0',
+      }"
+    >
+      <UCard
+        :ui="{
+          base: 'flex flex-col grow',
+          rounded: 'max-md:rounded-none',
+          header: { base: 'flex justify-between items-center gap-4' },
+          body: { base: 'grow', padding: '' },
+          footer: { base: 'sticky bottom-0 bg-white dark:bg-gray-900 rounded-b-[inherit]' },
+        }"
+      >
+        <template #header>
+          <h2
+            class="text-xl font-bold"
+            v-text="`Sales Items${selectedItems.length > 0 ? ` (${selectedItems.length} selected)` : ''}`"
           />
-          <UButton @click="addSaleItem">
-            Add
-          </UButton>
-        </UFormGroup>
-        <UFormGroup label="Sale Items">
-          <UTable
-            :columns="saleItemTableColumns"
-            :rows="saleItemTableRows"
-            :ui="{ td: { font: 'font-mono' } }"
-            @select="onSelectTableRow"
-          >
-            <template #selected-data="{ row }">
-              <UCheckbox :checked="row.selected.value" />
-            </template>
-            <template #image-data="{ row }">
+
+          <div class="flex items-center gap-2">
+            <UButton
+              v-if="saleItemTableRows.length"
+              :icon="isEditMode ? 'i-heroicons-pencil-solid' : 'i-heroicons-pencil'"
+              color="gray"
+              @click="toggleEditMode"
+            />
+
+            <UButton icon="i-heroicons-plus" @click="toggleAddItemModal" />
+          </div>
+        </template>
+
+        <UTable
+          v-model="selectedSaleItemTableRows"
+          :columns="saleItemTableColumns"
+          :rows="saleItemTableRows"
+        >
```

Comment:
Using `@select` would make the whole row clickable, which is easier on mobile though